src/lib/utils/validationSchemas.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
```

Comment:
great idea to extract the schema to a separate file! This improves code readability and modularity.